src/commands/queue/join.ts

```diff
@@ -89,9 +94,29 @@ const command: Command = {
         await client.utils.embeds.SimpleEmbed(interaction, { title: "Coaching System", text: queueData.getJoinMessage(user.id), empheral: true });
 
         // await client.utils.embeds.SimpleEmbed(interaction, "TODO", `Command \`${path.relative(process.cwd(), __filename)}\` is not Implemented Yet.`);
+        
+        await informCoaches(client, queueData);
     },
 };
 
+/**
+ * sends a private message to all coaches having an active session on the given queue
+ * @param queueData queue the user joined in
+ */
+async function informCoaches(client: Bot, queueData: ArraySubDocumentType<Queue>) {
+    const activeSessions = await SessionModel.find({ queue: queueData.id, active: true }).exec();
+    try {
+        for(const session of activeSessions) {
+            const user = await client.users.fetch(session.user);
+            const dmChannel = await user.createDM();
```

Comment:
If the tutor has dms disabled, this will throw an error. Maybe we should create a fallback for that? (e.g. define a tutor-queue-news channel or similar). I think it's fine for now though.